### PR TITLE
Documentation Upgrade

### PR DESCRIPTION
**Pull request by bluemode**
## Description
Give cool things cool pages.
Why can't I see a mew?
Help the chidren.

## Motivation and Context
I was waiting on my pizza to get here so I made a few changes. 

## How Has This Been Tested?
lol, no

## Screenshots (if appropriate):
Umm, I was adding screenshots...

## Types of changes
- [x] Awesome BLUEMODE change that WON'T upset @sebastienvercammen 
- [ ] Awesome BLUEMODE change that WILL upset @sebastienvercammen 

## Checklist:
- [x] I have updated the documentation accordingly.
- [x] Annoy Seb or Thunderfox to merge
- [ ] Order another pizza.
- [x] Be Awesome!

### CODE REVIEW

File: docs/first-run/hashing.md [hunk 1]
```diff
@@ -29,14 +29,26 @@ If you use a configuration file, use `hash-key: [YourHashKeyHere, YourSecondHash
...
 
-### How about `Hashing server is unreachable, it might be offline.`
+### And this? 
+```
+Hashing server is unreachable, it might be offline.
+```
 Hashing server is temporarily unavailable (possibly offline). This could be due to maintenance or server failure. Please checkout discord for more information is you start getting this error.
 
-### And this one? `Invalid or expired hashing key: %s. + api._hash_server_token`
+### And this?
```
Comment (sebastienvercammen) [inline docs/first-run/hashing.md]: You can remove all `### And this?` titles.

File: docs/first-run/Blinding.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Account Blinding 
+
+### As of May 21st 2017, Niantic has implemented a new kind of ban. This ban will hide rare pokemon and other features within the Pokemon Go app when using an account that is blind. 
+
+## What Do We Know?
```
Comment (sebastienvercammen) [inline docs/first-run/Blinding.md]: Do we want to capitalize all words in titles? I vote no: we didn't do this in all other files, and we're not an online clickbait "news" website.

File: docs/extras/Webhooks.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ PokeAlarm is an example of a script you can run to accept webhook data and send
 
 The POST request made by RocketMap will contain the following data for pokemon type webhooks:
 
-```
+```json
 {
    "message":{
       "disappear_time":1493734519,
```
Comment (sebastienvercammen) [inline docs/extras/Webhooks.md]: In this particular message, someone replaced `longitude` with stars, making it invalid JSON. Can you replace both latitude and longitude with `0`?

File: docs/extras/Custom-CSS.md [hunk 4]
```diff
@@ -2,7 +2,43 @@
...
+## CSS classes
+There are four different classes added to style individual map templates. 
+
+Use class .mapPage for styles in map.html.
+
+Use class .mobilePage for styles in mobile_list.html.
+
+Use class .statisticsPage for styles in statistics.html.
+
+Use class .statusPage for styles in status.html.
```
Comment (sebastienvercammen) [inline docs/extras/Custom-CSS.md]: Make this an unordered list w/ `*`.

File: docs/extras/Webhooks.md (hunk 3 above)
Comment (bluemode) [inline docs/extras/Webhooks.md]: done!

**Pull request merged**